### PR TITLE
Fix/UI weight

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?
- Apply correct weight to header buttons
- Apply correct alignment to history modal
- Reorder footer logos according to design

### CODE REVIEW

File: webapp/src/components/Footer/index.js [hunk 1]
```diff
@@ -76,11 +76,11 @@ const Footer = () => {
           <ListItem className={classes.listItem}>
             <ListItemIcon>
               <a
-                href="https://www.reddit.com/r/ProtonChain"
+                href="https://www.instagram.com/protonxpr"
```
Comment (leisterfrancisco) [inline webapp/src/components/Footer/index.js:79]: This is just reordering buttons according to design

File: webapp/src/components/Header/index.js [hunk 2]
```diff
@@ -266,16 +275,25 @@ const Header = memo(({ onDrawerToggle, routes }) => {
...
-          <ProtonLogoTestnet className={classes.imageSm} />
-        ) : (
-          <ProtonLogo className={classes.imageSm} />
-        )}
-        {mainConfig.isTestnet ? (
-          <ProtonLogoDesktopTestnet className={classes.imageMd} />
-        ) : (
-          <ProtonLogoDesktop className={classes.imageMd} />
-        )}
+        <Link
```
Comment (leisterfrancisco) [inline webapp/src/components/Header/index.js:278]: Please check if this component is correctly used

File: webapp/src/components/HistoryModal/styles.js [hunk 3]
```diff
@@ -45,14 +46,15 @@ export default theme => ({
     alignItems: 'center'
   },
   bodySecondary: {
-    overflow: 'scroll',
+    overflow: 'auto',
```
Comment (leisterfrancisco) [inline webapp/src/components/HistoryModal/styles.js:49]: in windows were producing wrong look and feel, for that reason, I changed it to auto, @adriexnet please let me know if there is no problem with this new configuration

File: webapp/src/components/Header/index.js (hunk 2 above)
Comment (xavier506) [reply webapp/src/components/Header/index.js:278]: @tetogomez

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by adriexnet — APPROVED** [review]

**Pull request merged**